packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts

```diff
@@ -86,10 +86,18 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
      * synchronizes all containers and waits for a summary that contains the last processed sequence number.
      * @returns the sequence number of the summary
      */
-     async function waitForSummary(): Promise<number> {
-        await provider.ensureSynchronized();
+     async function waitForSummary(this: any): Promise<number> {
```

Comment:
if you want to do it this way, please you appropriate typing. we should only use any in exceptional circumstances